### DISCUSSION TITLE
Optional disable of "Discussion answered" in profile

### DISCUSSION THREAD

**Discussion by EricKotato** [community/community#2881] [Discussions] [answered]
Is there a way to disable "Discussion answered" in my profile?

![](https://user-images.githubusercontent.com/2903496/101588957-8e7a8980-39f8-11eb-8e7a-96eb6f561a78.png)

**Comment by evi-liu** [comment]
What's the reasoning behind the desire to disable the `Discussions answered` section?

**Reply by EricKotato** [reply]
Personally I think that "Discussions answered" isn't for "Highlights" section, especially with a low number of discussions. Also you can disable this to keep profile more clean, especially when you have feature "Disable Arctic Vault Contributor badge".

**Comment by sobolevn** [comment]
+1 here, I really like minimalistic profile page. I really like that I can disable almost everything.
But, no way to hide "1 discussion answered" 😞

**Reply by Rubiuscoffee** [reply]
I think it looks  good, idk lul

**Comment by syauqi** [comment]
And then there is a bug. I delete my answer on the discussions but it keeps showing even my answer is deleted.

**Comment by MusfiqDehan** [comment, marked as answer]
Hey @evi-liu , 

Sorry for bothering you.

Is GitHub disabled the "Discussion Answered" badge? I have answered some discussions and got the "Galaxy Brain" Achievement but didn't get the "Discussion Answered" badge in my Highlights section.

**Reply by radar-blip** [reply]
The `Discussion answered` highlight was replaced by the Galaxy Brain Badge. So yes, Discussion answered highlight was disabled.

**Reply by EricKotato** [reply]
That's the good news. I guess it makes sense to mark topic as answered now.
